.gitignore

```diff
@@ -22,6 +22,9 @@ coverage
 # Compiled binary addons (http://nodejs.org/api/addons.html)
 build/Release
 
+# Idea
+.idea
+
```

Comment:
only adding this will not work :) you should remove from the commit all .idea files.

But nevertheless we should discuss the seed project to use (its architecture). So I will wait for a while and not merge this pull request